website/src/models/user.js

```diff
@@ -702,6 +704,30 @@ schema.methods.getGroups = function getUserGroups () {
   return userGroups;
 };
 
+schema.methods.sendMessage = async function sendMessage (userToReceiveMessage, messageData) {
+  let msg;
+  let sender = this;
+
+  if (!messageData.type) {
+    msg = messageData.message;
+  } else {
+    msg = `Hello ${userToReceiveMessage.profile.name }, ${sender.profile.name} has sent you `;
```

Comment:
@TheHollidayInn can you move this to a localized string? using the language in userToReceiveMessage.preferences.language